dns/zones/terraform.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_version = "~> 0.15.5"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "2.62.0"
+    }
+  }
+  backend "azurerm" {
+  }
+  experiments = [module_variable_optional_attrs]
```

Comment:
I don't think this is required (also for dns/records)

Reply:
removed experiments (and retested without)